sources/packages/web/src/assets/css/global-style-variables.scss

```diff
@@ -42,6 +42,7 @@ $formio-content: 16px;
 $formio-content-lh: 24px;
 $formio-header-margin: 0.5rem;
 $font-size-content: 16px;
+$font-size-small: 14px;
```

Comment:
is this really needed? can you double-check with @hellolynn-tbtb. we use 16px for content accrosss the project

Reply:
yes checked with her